Mathlib/Data/NNRat/Defs.lean

```diff
@@ -365,10 +367,14 @@ theorem natAbs_num_coe : (q : ℚ).num.natAbs = q.num := rfl
 
 lemma coprime_num_den (q : ℚ≥0) : q.num.Coprime q.den := by simpa [num, den] using Rat.reduced _
 
--- TODO: Rename `Rat.coe_nat_num`, `Rat.intCast_den`, `Rat.ofNat_num`, `Rat.ofNat_den`
```

Comment:
Sorry, why are you removing this TODO? Here's the correct updated version
```
-- TODO: Rename `intCast_num`, `intCast_den`, `ofNat_num`, `ofNat_den`, `inv_natCast_num`,
-- `inv_natCast_den`, `inv_intCast_num`, `inv_intCast_den`
```

Reply:
Surely `ofNat_num` and the `intCast` ones shouldn't be in that list as they doesn't exist?

Reply:
https://leanprover-community.github.io/mathlib4_docs/Std/Data/Rat/Lemmas.html#Rat.ofNat_num ?

Reply:
Oh I see the TODO is in the wrong file. Can you move it `Data.Rat.Defs`

Reply:
I've just put the TODO back; if you want to clarify it further, please do so in your own PR.